lua/cfcfactions/core/cl/cl_clientstartup.lua

```diff
@@ -44,27 +44,35 @@ function cfcFactions:DisplayMenu()
 end
 
 --If the defined key is properly set, users can use that specific key to also open/close the derma
-hook.Add("PlayerButtonDown", "CFC_FAC_MENUKEYDOWN", function(ply, button)
+local function menuKeyDown(ply, button)
     if input.GetKeyName(button) == cfcFactions.Config.CLIENT_KEY then
         cfcFactions:DisplayMenu()
     end
-end)
+end
+
+hook.Add("PlayerButtonDown", "CFC_FAC_MENUKEYDOWN", menuKeyDown)
 
 --If a user requests to display the derma from serverside
-net.Receive('CFC_Fac_ToggleDerma', function(len, ply)
+local function toogleDerma(len, ply)
```

Comment:
These method names seem a little to vague for me. Best practice is usually to scope them by name in some way.


```suggestion
local function toggleFactionsDerma(len, ply)
```
(You'll need to change the other reference to it, of course)